src/6model/bootstrap.c

```diff
@@ -586,6 +586,7 @@ static void string_consts(MVMThreadContext *tc) {
     string_creator(path, "path");
     string_creator(config, "config");
     string_creator(replacement, "replacement");
+    string_creator(dot, "dot");
```

Comment:
Shouldn't this be `string_creator(dot, ".");`?

Reply:
Yes. It should. Fixed.